.docker/entrypoint/spec/commander_spec.rb

```diff
@@ -1,16 +1,17 @@
 # frozen_string_literal: true
 
 require 'commander'
-require 'spec_logger'
+require 'helpers/spec_logger'
 require 'jekyll'
 require 'docker_image'
 require 'jekyll_environment'
 
 describe Jekyll::PlantUml::Commander do
   let(:version) { '0.0.1-test.0' }
   subject(:commander) do
+    data_dir = File.join(__dir__, 'data')
     Jekyll::PlantUml::Commander.new(
-      Jekyll::PlantUml::JekyllEnvironment.new('development', __dir__, __dir__),
+      Jekyll::PlantUml::JekyllEnvironment.new('development', data_dir, data_dir),
```

Comment:
Codacy found an issue: [Line is too long. [81/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5768825)